assets/js/common/CheckableWarningMessage/CheckableWarningMessage.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Input from '@common/Input';
+import { noop } from 'lodash';
+import { EOS_WARNING_OUTLINED } from 'eos-icons-react';
+
+function CheckableWarningMessage({
+  hideCheckbox = false,
+  warningText,
+  checked,
+  onChecked = noop,
+}) {
+  return (
+    <div className="flex items-center border border-yellow-400 bg-yellow-50 p-4 rounded-md text-yellow-600 mb-4">
```

Comment:
Maybe adding `text-sm` and `font-normal` makes it closer to the mock up we have.
@jagabomb what do you think?

Reply:
I would apply the suggested changes for now we can revisit this @jagabomb